aesara/tensor/math.py

```diff
@@ -1154,14 +1154,10 @@ def round_half_away_from_zero(a):
 
 
 @scalar_elemwise
-def sqr(a):
+def square(a):
```

Comment:
Before removing `sqr` completely, I suggest adding some deprecation warnings and testing those warnings. Once we are sure that no one uses `sqr`, we can then safely remove it.